dms-presentation/src/main/kotlin/team/aliens/dms/domain/manager/dto/request/AccountId.kt

```diff
@@ -0,0 +1,6 @@
+package team.aliens.dms.domain.manager.dto.request
+
+object AccountId {

```

Comment:
```suggestion
object AccountIdRegex {
```
Password도 같이 변경하면 좋을 것 같아요